### DISCUSSION TITLE
Switch - Visual Accessibility Improvement

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#1551] [Proposals]
### Goal

Según los principios de las WCAG para el componente Switch, uno de los criterios que forma parte del principio Perceptible es que el color no se utilice como único medio para transmitir su estado o distinguir elementos visuales.

Haciendo bench sobre cómo se trata este componente en otros sistemas de diseño, veo que muchos DS ya incluyen esta evolución para cumplir este requisito.

Algunos ejemplos:
 - [Material 3](https://m3.material.io/components/switch/accessibility#c0e9fae1-48df-428b-b028-4f7be071ada3)
 - [iOS](https://developer.apple.com/design/human-interface-guidelines/toggles)
 - [Atlassian](https://atlassian.design/components/toggle/examples)
 - [Carbon](https://carbondesignsystem.com/components/toggle/usage/#default-toggle)

### Context

Nuestros switch actualmente no cumplen este requisito dado que el "track" y el "thumb" pueden no ser claros para personas con problemas de visión dado que el estado del switch solo se identifica a través del color y la forma del elemento.

### Impact

Mejorar la accesibilidad de los switches puede tener un impacto significativo en la UX y en la cantidad de usuarios a los que los productos pueden puede llegar.

### Discussion Questions

.

### Do you have a visual proposal?

https://www.figma.com/file/J0LzwXn0tzGNIkgvFnt0Yz/branch/3NXryrP7s0L4JmELqt93c7/%F0%9F%94%B8-Switch-Component-Specs?type=design&node-id=704%3A3039&mode=design&t=9ynHMGCIiFLjoKqv-1

**Comment by aweell** [comment]
Will be solved with:

https://github.com/Telefonica/mistica-design/issues/1918 
https://github.com/Telefonica/mistica-design/issues/1917
